dev/devicelab/bin/tasks/ios_content_validation_test.dart

```diff
@@ -126,17 +138,35 @@ Future<void> main() async {
         if (!outputAppFrameworkBinary.existsSync()) {
           fail('Failed to re-embed ${outputAppFrameworkBinary.path}');
         }
-      });
 
-      if (foundProjectName) {
```

Comment:
Moved these failures to where they are detected, instead of the end of the test.